sdk/inc/azure/core/az_mqtt5_rpc_server.h

```diff
@@ -165,9 +165,9 @@ typedef struct
    */
   az_span content_type;
   /**
-   * @brief The service id of the request.
+   * @brief The service model identifier of the request.
```

Comment:
```suggestion
   * @brief The model identifier of the request.
```
is the keyword service still relevant here?

Reply:
Wording is based on the design descriptions, service model is still used